core/input/input.cpp

```diff
@@ -1495,6 +1495,7 @@ void Input::parse_mapping(const String &p_mapping) {
 		JoyAxis output_axis = _get_output_axis(output);
 		if (output_button == JoyButton::INVALID && output_axis == JoyAxis::INVALID) {
 			print_verbose(vformat("Unrecognized output string \"%s\" in mapping:\n%s", output, p_mapping));
+			continue;
```

Comment:
This fixes what I reported here: https://github.com/godotengine/godot/pull/73224#discussion_r1587356111

![image](https://github.com/godotengine/godot/assets/4701338/81e6ca45-82c7-4f0b-8b8f-0b92de29ec07)

CC @rsubtil

Reply:
Yes, it makes sense, I didn't notice that detail when changing the error message to a warning :+1:

Reply:
I believe this should also fix these issues:
- #88630
- #92578